### PR TITLE
Additions to main Portfolio repo

### PR DESCRIPTION
**Pull request by mmatongo**
This pull request contains additions to the main portfolio repo.
The changes include;

- `linters.yml` - This is the linter file responsible for code checking.
- `readme.md` - General documentation and acknowledgment.
- `styles.css` - General styling for the *.html.
- `/bg` - Contains all necessary background files.
- `/icons` Contains all icon files.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      </nav>
+      <button type="button" class="btn">
+        <i class="material-icons">menu</i>
+      </button>
+    </header>
+    <section id="headline" class="background">
+      <section class="intro">
+        <h1>
+          <span>Hey there, I’m Daniel</span>
+          <span class="title"> I’m a software developer</span>
```
Comment (Lameck1) [inline index.html:28]: Kindly refactor this to a ```<p>``` tag and outside the ```<h1>``` tag as per the requirement.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        /></a>
+        <a href="#"
+          ><img class="icon" src="icons/medium.svg" alt="medium logo"
+        /></a>
+        <a href="#"
+          ><img class="icon" src="icons/github.svg" alt="github logo"
+        /></a>
+        <a href="#"
+          ><img class="icon" src="icons/angellist.svg" alt="angellist logo"
+        /></a>
```
Comment (Lameck1) [inline index.html:54]: Consider refactoring these icons to a ```<ul>``` tag and each icon into a single ```<li>``` tag.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi @mmatongo, 

Good job so far!
There are some issues that we still need to work on though.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (mmatongo) [reply index.html:54]: Is this optional?

File: index.html (hunk 1 above)
Comment (mmatongo) [reply index.html:28]: is this optional?

**Comment by Lameck1** [comment]
No, it's not. Any comment with the [OPTIONAL] tag, is the one considered
optional.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# 👏 

hi @mmatongo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**